src/pyfaf/storage/generic_table.py

```diff
@@ -49,38 +49,31 @@ def has_lob(self, name):
 
     # lob for Large OBject
     # in DB: blob = Binary Large OBject, clob = Character Large OBject
-    def get_lob(self, name, binary=True):
+    def get_lob(self, name):
         lobpath = self.get_lob_path(name)
 
         if not os.path.isfile(lobpath):
```

Comment:
Unrelated: I think we should just try to `open()` the thing and handle errors from there.